test-data/unit/check-modules.test

```diff
@@ -131,9 +131,9 @@ def f() -> None: pass
 [case testImportWithinClassBody2]
 import typing
 class C:
-    from m import f
+    from m import f  # E: Method must have at least one argument
```

Comment:
This seems to be placed on the wrong line.

Reply:
This is actually on the correct line. In this kind of situation, `C.f` is capable of working like a method would. However, because this `f` takes no arguments, if you try to do `C().f()` you'd get a TypeError at runtime.

I agree that for the code in this test, where there is no attempt to use `C.f` as a method, the error is a little surprising. But I think the code in this test is not particularly idiomatic. Maybe we could make it only error if you try to call it as a method, but implementing that is not easy. So I'd like to keep that out of scope for this PR, which fixes crashes and whose behaviour is technically correct.

Reply:
Oh, I see! 👍